database/factories/ModelFactory.php

```diff
@@ -83,7 +83,6 @@
          */
         'school_id' => $this->faker->optional()->school_id,
         'source' => 'phpunit',
-        'referrer_user_id' => $this->faker->optional()->northstar_id,
```

Comment:
Is this to avoid triggering the c.io event sporadically unless we explicitly want to test?

Reply:
Exactly! We'd have had to add the mocks into a lot more tests -- I think it's better to explicitly test that we only call GraphQL when it's a referral post.